src/entry.js

```diff
@@ -21,7 +21,7 @@ const render = () => {
   )
 }
 
-if (window && window.localStorage) window.localStorage.clear()
+// if (window && window.localStorage) window.localStorage.clear()
```

Comment:
yeah, i think we should do the prod/dev check, and then maybe in dev, have an evergreen button that can clear it if necessary

for now, will uncomment, and then we can merge